sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample11_MultiCategoryClassifyAsync.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    public partial class TextAnalyticsSamples : SamplesBase<TextAnalyticsTestEnvironment>
+    {
+        [Test]
+        public async Task MultiCategoryClassifyAsync()
+        {
+            // Create a text analytics client.
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+
+            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Get input document.
+            string document = @"I need a reservation for an indoor restaurant in China. Please don't stop the music. Play music and add it to my playlist.";
+
+            // Prepare analyze operation input. You can add multiple documents to this list and perform the same
+            // operation to all of them.
+            var batchDocuments = new List<TextDocumentInput>
+            {
+                new TextDocumentInput("1", document)
+                {
+                     Language = "en",
+                }
+            };
+
+            // Set project and deployment names of the target model
+#if SNIPPET
```

Comment:
looks like there is no snippet defined in this class, so this section of code is not necessary. you could just do 
> string projectName = TestEnvironment.MultiClassificationProjectName;
            string deploymentName = TestEnvironment.MultiClassificationDeploymentName;

Reply:
Same for the other 3 sample classes

Reply:
Sure thing, removed them.

I had them there because I remember in DT there were some samples where we wanted to have the `#if SNIPPET` directives for clarity, even when the code was not actually used in a snippet.

Reply:
Fair. DT was more complex as it required function calls to get the storage setup. here it is just a string so user can follow easier